packages/faceted-search/src/components/AdvancedSearch/AdvancedSearch.scss

```diff
@@ -10,35 +11,30 @@ $error-color: #BA4D51;
 
 	.adv-search-filter-icon {
 		position: absolute;
-		color: $dark-silver;
-		top: $padding-large;
-		left: $padding-large;
+		color: tokens.$coral-color-neutral-icon-weak;
+		top: tokens.$coral-spacing-m;
+		left: 1.8rem;
```

Comment:
That seems random!

Reply:
It's almost magic yes but it's required for the alignment of the icon

Reply:
Then ideally a `$iconAlignmentOffset` var could help male sense of the magic number :)